Source/DafnyCore/Verifier/Translator.cs

```diff
@@ -8296,17 +8296,17 @@ public ForceCheckToken(IToken tok)
       }
     }
 
-    Bpl.Expr trTypeParamOrOpaqueType(TopLevelDecl x, List<Type>/*?*/ tyArguments = null) {
-      Contract.Requires(x is TypeParameter || x is OpaqueTypeDecl);
+    Bpl.Expr trTypeParamOrAbstractType(TopLevelDecl x, List<Type>/*?*/ tyArguments = null) {
```

Comment:
Please capitalize `trTypeParamOrAbstractType` to `TrTypeParamOrAbstractType`, since you're editing it anyway. And while you're at it, would you mind also renaming method `nameTypeParam` to `NameTypeParam` just above it?

Reply:
Changed both